django/README.md

```diff
@@ -0,0 +1,107 @@
+# Django + Celery Sample App
+
+This example shows how to leverage [Okteto](https://okteto.com) to develop a Django + Celery Sample App directly in Kubernetes. The Django + Celery Sample App is deployed using raw Kubernetes manifests.
+
+Okteto works in any Kubernetes cluster (local or remote) by reading your local Kubernetes credentials.
+
+## Step 1: Install the Okteto CLI
+
+Install the Okteto CLI by following our [installation guides](https://github.com/okteto/okteto/blob/master/docs/installation.md).
+
+
+## Step 2: Django + Celery Sample App
+
+Clone the repository and go to the java-kubectl folder.
+
+```console
+git clone https://github.com/okteto/samples
+cd samples/django
+```
+
+Deploy the Django + Celery Sample App by using the following command:
+```console
+kubectl apply -f manifests
+statefulset.apps "cache" created
+service "cache" created
+statefulset.apps "db" created
+service "db" created
+statefulset.apps "queue" created
+service "queue" created
+deployment.apps "web" created
+service "web" created
+deployment.apps "worker" created
+```
+
+Check that all pods are running ok. Now you can access you app, the easiest way is to execute:
+
+```
+kubectl port-forward $(kubectl get pod -l app=web -o jsonpath="{.items[0].metadata.name}") 8080:8080
```

Comment:
This will disconnect on the 'okteto up'. Maybe do the port-forward to the service instead?